docs/curriculum-3.0.1.2.openapi.yaml

```diff
@@ -36,6 +36,10 @@ components:
                 pattern: >-
                   ^(https?:\/\/)([a-zA-Z0-9-]{1,63}\.)+[a-zA-Z-]{2,24}(:[0-9]{1,5})?(\/(([\w\-\.~]|%[a-fA-F0-9]{2}|[!\$&'\(\)\*\+,;=:@])*\/?)*)?
                 type: string
+              description:
+                description: The Topic overview paragraph
+                pattern: ^[ -~]{0,256}$
```

Comment:
the range character ``-~``  -~ matches a single character in the range between   (index 32) and ~ (index 126) (case sensitive)
So this is a single line up to 256 character long, with no special characters (cr, tab, lf, etc.)

Reply:
The openapi spec's are all generated, you'd be better off looking at the mongo collection schema files and test data.

Reply:
I understand now. I haven't tested it myself yet but it would be neat if Mongo supported unicode character classes yo do something like this
```
/[\p{ASCII}--\p{Other}]/v
```

Reply:
Please open an issue on this repo  when you've concluded your research - those patterns are from that project.